### ISSUE TITLE
Drop-down boxes clash with autofill

### ISSUE THREAD

**Issue by olzama** [delph-in/matrix#556]
<img width="484" alt="Screen Shot 2021-01-12 at 10 45 02 AM" src="https://user-images.githubusercontent.com/10963114/104358525-6d0f3080-54c3-11eb-9e87-d5713ebbe2bc.png">

**Comment by emilymbender** [comment]
I guess this is more than just annoying if the auto-fill info is PII (names, phone numbers, etc) so worth figuring out so we aren't inadvertently storing such info!

**Comment by olzama** [comment]
Good point. I assumed it was just my Chrome, not us (filling that in), but I have no idea, actually.

**Comment by olzama** [comment]
I do think that in general, we can assume that Chrome is responsible for handling its own autofill. Presumably it has privacy in place. That being said, we should look into disallowing any autofill that has nothing to do with our page; surely that should be possible.
